libraries/aesthetic/src/main/java/com/afollestad/aesthetic/Aesthetic.java

```diff
@@ -78,7 +78,7 @@
 
   @SuppressLint("CommitPrefEdits")
   private Aesthetic(Context context) {
-    this.context = context;
+    this.context = context.getApplicationContext();
```

Comment:
This should not be necessary, as long as we're clearing the context reference in `onPause()`